### DISCUSSION TITLE
Disable LSP (formatting?) completely for filetype

### DISCUSSION THREAD

**Discussion by mikker** [LazyVim/LazyVim#603] [Q&A]
Whenever I save a file in my Vimwiki (markdown) I get a notification saying:

```
[LSP] Format request failed, no matching language servers.
```

Can I disable auto-formatting completely for a certain filetype, in this case `vimwiki`?

**Comment by dpetka2001** [comment]
See this [issue](https://github.com/LazyVim/LazyVim/issues/575), the author has an autocommand for what you want I believe.

**Reply by mikker** [reply]
I couldn't translate that into something useful. Could you help me one more step of the way?

**Reply by dpetka2001** [reply]
What exactly did you do and did not work? Did you put the code for the autocmd in `~/.config/nvim/config/autocmds.lua`?
